hedera-node/src/test/java/com/hedera/services/state/submerkle/ExpirableTxnRecordTest.java

```diff
@@ -504,13 +502,7 @@ void v0230DeserializeWorksWithAllowanceMaps() throws IOException {
 		given(fin.readLong())
 				.willReturn(subject.getFee())
 				.willReturn(subject.getExpiry())
-				.willReturn(subject.getSubmittingMember())
-				.willReturn(ownerNum.longValue())
-				.willReturn(spenderNum.longValue())
-				.willReturn(initialAllowance)
-				.willReturn(ownerNum.longValue())
-				.willReturn(initialAllowance)
-				.willReturn(ownerNum.longValue());
+				.willReturn(subject.getSubmittingMember());
```

Comment:
slightly concerned about mocking the written bytes. Can we do the same thing as MerkleAccountStateTest ?

Reply:
Done.